lib/background_worker.rb

```diff
@@ -30,6 +30,12 @@ def self.verify_active_connections!
     end
   end
 
+  def self.release_connections!
+    if defined?(ActiveRecord) && ActiveRecord::VERSION::MAJOR == 4
+      ActiveRecord::Base.clear_all_connections!
```

Comment:
Actually what happens with rails 5, should we instead say:
if ActiveRecord::Base.respond_to? :clear_all_connections